query-api/src/test/resources/projection/schema-test-projection-single-match-single-project.json

```diff
@@ -6,7 +6,7 @@
         "op": "$eq",
         "rfield": "someuser"
     },
-    "project": {
+    "projection": {
```

Comment:
Accept both project and projection. Keep "project" section, and add "projection" with same contents